recipe/meta.yaml

```diff
@@ -32,6 +47,19 @@ requirements:
     - libgrpc {{ libgrpc }}
 
   host:
+    - cudnn             # [cuda_compiler_version != "None"]
+    - nccl              # [cuda_compiler_version != "None"]
+    - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
```

Comment:
this isn't needed - the cuda activation feedstock injects this https://github.com/AnacondaRecipes/cuda-nvcc-feedstock/blob/afd127182f47170fe25d9b58e14bcbf11a795709/recipe/meta.yaml#L45